portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -10,20 +10,17 @@
     private String name; // name of the commenter
     private String content; // content of the comment
     private Date date; // date the comment was left
-    private String email;
 
-    public Comment(String name, String content, String email) {
+    public Comment(String name, String content) {
         this.name = name;
         this.content = content;
         this.date = new Date();
-        this.email = email;
     }
 
-    public Comment(String name, String content, Date date, String email) {
+    public Comment(String name, String content, Date date) {
         this.name = name;
         this.content = content;
         this.date = date;
-        this.email = email;
```

Comment:
Are these the 'old' things you mentioned. I would pull master into this branch before merging this back into master so you don't accidentally break your code or lose functionality.

Reply:
I merged after the PR, so all this stuff is already gone